src/js/components/Accordion/__tests__/__snapshots__/Accordion-test.js.snap

```diff
@@ -224,6 +224,11 @@ exports[`Accordion AccordionPanel 1`] = `
         onMouseOut={[Function]}
         onMouseOver={[Function]}
         role="tab"
+        style={
```

Comment:
I think we should avoid passing `style` if we aren't setting `zIndex`.

Reply:
Good catch! fixed.